src/grpc.ts

```diff
@@ -126,9 +127,9 @@ export class GrpcClient {
    * @constructor
    */
   constructor(options: GrpcClientOptions = {}) {
+    options.useJWTAccess = true;
```

Comment:
I think we ended up making these changes in the generator and this was accidentally left in.